### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by trav3711**
Project import generated by Copybara.

### CODE REVIEW

File: tools/guid_generator/ontology/types_guid_generator/types_guid_generator.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 Google LLC
```
Comment (jmact17) [inline tools/guid_generator/ontology/types_guid_generator/types_guid_generator.py:1]: is it possible to pull in the updates to this file (and tests) from my recent cl?

Comment (trav3711) [reply tools/guid_generator/ontology/types_guid_generator/types_guid_generator.py:1]: Done.

File: .github/workflows/type_guid_generator.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# This workflow adds guids to the entity types ontology
```
[the remark this answers is not in the document]
Comment (trav3711) [reply .github/workflows/type_guid_generator.yml]: Done.

**Review by jmact17 — APPROVED** [review]

**Review by charbull — APPROVED** [review]

**Review by azenla — APPROVED** [review]
LGTM

**Pull request merged**